ksql-engine/src/main/java/io/confluent/ksql/topic/DefaultTopicInjector.java

```diff
@@ -80,15 +84,26 @@ public DefaultTopicInjector(
     final PreparedStatement<? extends CreateAsSelect> cas =
         (PreparedStatement<? extends CreateAsSelect>) statement;
 
+    final String prefix =
+        propertyOverrides.getOrDefault(
```

Comment:
Was it a bug that this code previously wasn't checking for the possibility of `KSQL_OUTPUT_TOPIC_NAME_PREFIX_CONFIG` being overridden in the property overrides, or did something else change to make this check necessary now? If the former, we should maybe add a test to ensure this isn't forgotten again in the future.

Reply:
It was indeed a bug (I caught this because now QTT goes through this code path). I will add an explicit test for it.